server/services/nextcloud-talk/index.js

```diff
@@ -0,0 +1,41 @@
+const logger = require('../../utils/logger');
+const { ServiceNotConfiguredError } = require('../../utils/coreErrors');
+const MessageHandler = require('./lib');
+
+module.exports = function NextcloudTalkService(gladys, serviceId) {
+  const axios = require('axios');
+
+  const messageHandler = new MessageHandler(gladys, serviceId, axios);
+  /**
+   * @public
+   * @description This function starts the Nextcloud Talk Service
+   * @example
+   * gladys.services.nextcloudTalk.start();
+   */
+  async function start() {
+    logger.info('Starting Nextcloud Talk service');
+    const tokens = await gladys.variable.getVariables('NEXTCLOUD_TALK_TOKEN', serviceId);
+
+    if (!tokens || tokens.length === 0) {
+      throw new ServiceNotConfiguredError('No Nextcloud token found. Not starting Nextcloud Talk service');
+    }
+    await messageHandler.connect(tokens);
+  }
+
+  /**
+   * @public
+   * @description This function stops the Nextcloud Talk Service
+   * @example
+   * gladys.services.nextcloudTalk.stop();
+   */
+  async function stop() {
+    logger.info('stopping Nextcloud Talk service');
+    await messageHandler.disconnect();
+  }
+
```

Comment:
Oh, and just one other feedback, can you implement the new function `isUsed` ?

It should return true/false if the user is actively using the service :) 

Like: 

https://github.com/GladysAssistant/Gladys/blob/master/server/services/zigbee2mqtt/index.js#L38